base/array.jl

```diff
@@ -1129,6 +1129,21 @@ function pop!(a::Vector)
     return item
 end
 
+function pop!(a::Vector, i::Integer)
+    x = a[i]
+    _deleteat!(a, i, 1);
+    x
+end
+
+pop!(a::Vector, i::Integer, default) =
```

Comment:
Boring comment but this looks like it should be a full length function?

Reply:
Ok